daffodil-core/src/main/scala/org/apache/daffodil/dsom/SequenceGroup.scala

```diff
@@ -358,6 +358,8 @@ final class ChoiceBranchImpliedSequence(rawGM: Term)
   extends SequenceTermBase(rawGM.xml, rawGM.optLexicalParent, rawGM.position)
   with SequenceDefMixin {
 
+  override final lazy val groupMembers: Seq[Term] = LV('groupMembers) { Seq(rawGM) }.value
```

Comment:
does not need to be an LV. It's just taking a constructor arg. No error SDE can happen here that we want blamed/positioned  to this object, nor can anything "circular" happen. Those are the two reasons to make something an LV.

Reply:
Fixed.